llvm/lib/Target/AIE/AIE2GenInstrInfo.td

```diff
@@ -94,6 +94,8 @@ let Itinerary = II_NE in {
 }
 let Itinerary = II_OR in {
   def OR : AIE2_alu_r_rr_inst_alu<0b0101, (outs eR:$mRx), (ins eR:$mRx0, eR:$mRy), "or", "$mRx, $mRx0, $mRy">;
+  let isCodeGenOnly = 1 in 
+    def MOV_OR : AIE2_alu_r_rr_inst_alu_mov_gpr<0b0101, (outs eR:$mRx), (ins eR:$mRx0), "or", "$mRx, $mRx0, $mRx0">;
```

Comment:
I wouldn't mind defining this directly in terms of AIE2_inst_alu_instr32, computing the alu image here.

Reply:
Hi @martien-de-jong kindly clarify on what you mean by "alu image". Thank you.

Reply:
`let alu = {mRx0,mRx,mRx0,op,0b1};` as used in `AIE2_alu_r_rr_inst_alu_mov_gpr `

Reply:
I would like to be consistent with how all the other instructions are defined in the file.
If you think exposing `alu` image here is better, I will update it :).